config/services/dataplane_v1beta1_openstackdataplaneservice_ceph_hci_pre.yaml

```diff
@@ -5,3 +5,6 @@ metadata:
 spec:
   label: ceph-hci-pre
   playbook: osp.edpm.ceph_hci_pre
+  hasTLSCerts: True
+  issuers:
+    default: osp-rootca-issuer-internal
```

Comment:
do we have evidence in CI of this part? Also, I'm wondering if the plan is to generate certificates per service or we can just reuse the existing certs created for the edpm nodes.

Reply:
The parameter name has changed.  The new name is tlsCertsEnabled  as seen in https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml  or https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_libvirt.yaml

Reply:
Other than that.  This should be fine.